File: Easy/1979. Find Greatest Common Divisor of Array/ans.js
function gcd(a, b) {
  while (a != b) {
    if (a > b) {
      a -= b;
    } else {
      b -= a;
    }
  }
  return a;
}

var findGCD = function (nums) {
  var minValue = Math.min(...nums);
  var maxValue = Math.max(...nums);
  return gcd(maxValue, minValue);
};

File: Easy/1979. Find Greatest Common Divisor of Array/ans.cpp
#include<bits/stdc++.h>
using namespace std;

int findGCD(vector<int>& nums) {
    //  auto [mn, mx] = minmax_element(begin(nums), end(nums));
        int maxi = *max_element(nums.begin(),nums.end());
        int mini = *min_element(nums.begin(),nums.end());
        return gcd(maxi,mini);
    }

int main(){
    int n;
    cin>>n;
    vector<int>nums(n);
    for(int i=0;i<n;i++){
        cin>>nums[i];
    }
    cout<<findGCD(nums);
}